atomisticparsers/gromacs/parser.py

```diff
@@ -84,26 +86,21 @@ def str_to_input_parameters(val_in):
             return parameters
 
         def str_to_energies(val_in):
-            energy_keys_re = re.compile(r'(.+?)(?:  |\Z| P)')
-            keys = []
-            values = []
-            energies = dict()
-            for val in val_in.strip().split('\n'):
-                val = val.strip()
-                if val[0].isalpha():
-                    keys = [k.strip() for k in energy_keys_re.findall(val)]
-                    keys = ['P%s' % k if k.startswith('res') else k for k in keys if k]
-                else:
-                    values = val.split()
-                    for n, key in enumerate(keys):
-
-                        if key == 'Temperature':
-                            energies[key] = float(values[n]) * ureg.kelvin
-                        elif key.startswith('Pres'):
-                            key = key.rstrip(' (bar)')
-                            energies[key] = float(values[n]) * ureg.bar
-                        else:
-                            energies[key] = float(values[n]) / MOL * ureg.kJ
+            thermo_common = [r'Total Energy', r'Potential', r'Kinetic En.', r'Temperature',
+                             r'Pressure \(bar\)', r'LJ \(SR\)', r'Coulomb \(SR\)', r'Proper Dih.']
+            n_chars_val = re.search(rf'( +{"| +".join(thermo_common)})', val_in)
```

Comment:
I think the re pattern is wrong, we do not want to eat into the spaces after the key. should only be "|".join(thermo_common)